cluster/addons/registry/registry-pv.yaml.in

```diff
@@ -0,0 +1,16 @@
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: kube-system-kube-registry-pv
+  labels:
+    kubernetes.io/cluster-service: "true"
+spec:
+{% if pillar.get('cluster_registry_disk_type', '') == 'gce' %}
```

Comment:
I wasn't sure what the best way to do this is. This template should only be evaluated if the cluster registry is enabled which should only be permitted if one of the branches here is taken (for non-gce, they'd need to have their own config branch). If somehow this is evaluated otherwise, that's a bug and we should see a failure.